client/scripts/views/pages/view_proposal/body.ts

```diff
@@ -750,12 +790,41 @@ export const ProposalBodyText: m.Component<{
           ) {
             return getPlaceholder();
           }
-          return m(QuillFormattedText, { doc });
+
+          return m('.wrap', [
```

Comment:
@alexyoung23j Let's rename these classes to follow convention: that which is returned should be classed human-readably for what is being returned. So in this case, something like `.show-more-wrap` rather than just `.wrap`. otherwise this PR looks good